### PR TITLE
MM5-6146 added translations for search and filters

### PR DESCRIPTION
**Pull request by YevheniiKravchenko**

### CODE REVIEW

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 1]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_asset_relation_type {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_ASSET_RELATION_TYPE'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Asset Relation Type'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:743]:
```suggestion
      default_translation = 'Asset relation type'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 2]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_MEMBER'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'User'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Member'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:363]:
```suggestion
      default_translation = 'Bruger'
```

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 3]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_BUSINESS_WORKFLOW_INSTANCE'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Workflow instance'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Workflow-instans'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:523]: Task?

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 4]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_analytics_dashboard {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_ANALYTICS_DASHBOARD'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Analytics Dashboard'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:615]:
```suggestion
      default_translation = 'Analytics dashboard'
```

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 5]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_CONFIGURATION_FIELD_VALUE'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Configuration field value'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Konfigurations-felts-værdi'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:715]:
```suggestion
      default_translation = 'Konfigurations-felt-værdi'
```

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 6]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_ASSET_RELATION_TYPE'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Asset relation type'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Assetrelationstype'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:747]:
```suggestion
      default_translation = 'Asset-relations-type'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl (hunk 3 above)
Comment (YevheniiKravchenko) [reply MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:523]: I don't know, originally on backend this filter calls 'Business Workflow Instance'

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 7]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+  ]
+}
+
+resource configservice_label system_audit_trail_filter_business_workflow_instance {
+  key = 'SYSTEM_AUDIT_TRAIL_FILTER_BUSINESS_WORKFLOW_INSTANCE'
+  group = 'System Audit Trail'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Workflow instance'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:519]:
```suggestion
      default_translation = 'Task'
```

File: MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl [hunk 8]
```diff
@@ -301,3 +301,451 @@ resource configservice_label system_audit_trail_view_change_history_tooltip {
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Workflow-instans'
```
Comment (Poiar) [inline MM/6.1.0/config/media_manager_5/labels/System Audit Trail.dcl:523]:
```suggestion
      default_translation = 'Task'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

**Review by Poiar — APPROVED** [review]

**Pull request merged**